django_kafka/registry.py

```diff
@@ -34,9 +35,16 @@ def get_key(self, cls: Type[T]) -> str:
 
     def register(self, cls: Type[T]):
         key = self.get_key(cls)
+        if key in self._classes:
+            raise DjangoKafkaError(f"`{key}` is already registered.")
         self._classes[key] = cls
 
 
+class ConnectorsRegistry(Registry["Connector"]):
+    def get_key(self, cls) -> str:
+        return cls().name
```

Comment:
Any alternative to access the `name` property from the class instead, to not initialize the class and start a session??

Reply:
will apply it to avoid unnecessary instance creation.